src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -310,6 +313,93 @@ public void terminatesMonitorsAndProcessIfClosedNormal() throws Exception {
     }
 
     /**
+     * Runs a java process which will throw a stack trace and makes sure it
```

Comment:
@dean-e-clark first word in Javadoc for test should be a class tested and second should be `can`